src/main/java/org/jbake/app/Oven.java

```diff
@@ -174,7 +167,7 @@ public void bake() {
                 }
         } finally {
                 db.close();
-//                Orient.instance().shutdown();
```

Comment:
Ah ok. Yes I can do that. I'm using jbake in combination with gradle and the continous task runner. Therefor I'm not aware of this breaking the watch mode for the cli.